src/functionalTest/kotlin/com/cognifide/gradle/aem/pkg/PackagePluginTest.kt

```diff
@@ -270,12 +273,13 @@ class PackagePluginTest : AemBuildTest() {
                 
                 repositories {
                     jcenter()
+                    maven("https://repo.adobe.com/nexus/content/groups/public")
                 }
                 
                 tasks {
                     packageCompose {
-                        nestPackageBuilt(":ui.content:packageCompose")
-                        installBundleBuilt(":ui.apps:bundleCompose")
+                        nestPackageProject(":ui.content")
```

Comment:
this will cause also running unit tests for installed bundle and package validation for nested package